src/token_response.ts

```diff
@@ -65,7 +65,7 @@ export const nowInSeconds = () => Math.round(new Date().getTime() / 1000);
 export class TokenResponse {
   accessToken: string;
   tokenType: TokenType;
-  expiresIn: number|undefined;
```

Comment:
This should continue to be a `number`. See comment below.

Reply:
Having this type as `number` will break the instantiation of the class because of the different types in [expiresIn](https://github.com/openid/AppAuth-JS/blob/master/src/token_response.ts#L77).